public/app/screens/main/navigation/navigation.jsx

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var React = require('react'),
+  Link = require('react-router').Link,
+  StyleMixin = require('mixins/style-mixin');
+
+var Navigation = React.createClass({
+  mixins: [StyleMixin(require("./style.scss"))],
+  render: function () {
+    // TODO: Use mixins instead of CSS classes
+    // TODO: Pass number of elements to mixin instead of specifying class name
+    var classes = "cc_navigation icon-bar two-up " + this.props.orientation;
```

Comment:
Classes `icon-bar two-up` should be included by `cc_navigation`? This could however by done by a different PR that concentrates on using mixins for this icon-bar